cfpb/cfgov-refresh
# New blog newsroom

Introduces the new Blog and Newsroom page types

## Additions

- BlogPage and NewsroomPage 
- Article Subheader

## Changes

- Changes some templates around
- Fixes some bugs
- Learn, DocDetail, Blog, Newsroom pages have an option to add an Article Subheader for blog introductory paragraphs
- Full Width Text now has a Media option for images

## Testing

- `cfgov/manage.py migrate` (make sure you have the local.py database settings configured, if not see Notes)
- Go under the blog page and create a new blog post!
 - Add Article Subheader to the Header streamfield
 - Add the media option in the Content streamfield
 - Add some sidefoot content
- Preview it, Save it, Publish, Check it out on the site!
- Repeat for NewsroomPage

## Review

- @kave 
- @jimmynotjim 
- @richaagarwal 
- @rosskarchner 

## Screenshots
![screen shot 2016-04-20 at 5 28 24 pm](https://cloud.githubusercontent.com/assets/1412978/14691097/5335a818-071d-11e6-8968-e7c06824d4de.png)

## Notes

- local.py settings for database
```
DATABASES = {
    'default': {
        'ENGINE': MYSQL_ENGINE,
        'NAME': os.environ.get('MYSQL_NAME'),
        'USER': os.environ.get('MYSQL_USER'),
        'PASSWORD': os.environ.get('MYSQL_PW', ''),
        'HOST': os.environ.get('MYSQL_HOST', ''),  # empty string == localhost
        'PORT': os.environ.get('MYSQL_PORT', ''),  # empty string == default
        'OPTIONS': {
               "init_command": "SET storage_engine=MYISAM",
        }   
    },
}
```
